README.md

```diff
@@ -1,23 +1,68 @@
+Note that some of the user stories are ambiguous. There are many possible and/or
+acceptable solutions to these problems. Use your best judgement to determine
+what an appropriate solution is. It would be helpful to also demonstrate or 
+explain how you came to your conclusions. You may be asked to explain your
+solutions if it's not immediately clear what your reasoning was. Additionally,
+you may be asked to pair with a Contently engineer to further expand on your
+work.
 
-User stories to implement
-----------------
- - A user should not be able to complete a task when creating it.
- - A user can filter their view by complete or incomplete tasks.
- - A user's default view is filtered to show only incomplete tasks.
- - A user should be able to keep multiple lists.
- - A user should have access to only their tasks. (may require devise or a creative, lightweight auth system)
- - A user viewing a task should see a history of all its changes. (task audit trail)
- - A user should be able to see a history of all completed tasks and associated actions. (app audit trail)
+## Instructions
 
-Cleanup tasks
-----------------
-- Consolidate the repeated code between the create and edit feature specs for tasks.
+- [Create either a public or private duplicate of this repository](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/duplicating-a-repository).
+- Time-box your efforts to 2 to 4 hours.
+- Setup application
+    - [ ] `$ ruby -v` should return 2.7.1, else install Ruby 2.7.1
+    - [ ] `$ bundle install`
+    - [ ] `$ bundle exec rake db:setup`
+    - [ ] `$ bundle exec rails s`
+- Solve for several or more of the following problems:
+    - [ ] A user should not be able to complete a task when creating it.
+    - [ ] A user can filter their view by complete or incomplete tasks.
+    - [ ] A user's default view is filtered to show only incomplete tasks.
+    - [ ] A user should be able to keep multiple lists.
+    - [ ] A user should have access to only their tasks. (may require devise or 
+        a creative, lightweight auth system)
+    - [ ] A user viewing a task should see a history of all its changes. 
+        (task audit trail)
+    - [ ] A user should be able to see a history of all completed tasks and 
+        associated actions. (app audit trail)
+- Considerations for the work you submitted:
+    - [ ] Code actually works.
+    - [ ] Application is bootable and usable.
+    - [ ] Unit, integration, feature, and or end-to-end testing considered.
+    - [ ] Code is of sufficient quality and tidiness.
+    - [ ] Includes documentation on how to utilize new features / updates.
+- Commit your work with appropriate and informative git commit messages.
+- Push your work up to your duplicated git repository (on Github).
+- If your repository is private, give the following users access:
+    - [ ] [@jamesconant](https://github.com/jamesconant/)
+    - [ ] [@skipteel](https://github.com/skipteel)
```

Comment:
Any reason not to open this up to other engineers?

Reply:
For sure - I def think we want all eyes on this. I think I was thinking at the time we could add them on once we were tagged, but good point. Might as well just have them add.